app/sidekiq/central_mail/submit_central_form686c_job.rb

```diff
@@ -68,10 +67,10 @@ def upload_to_lh
 
     def get_files_from_claim
       # process the main pdf record and the attachments as we would for a vbms submission
-      form_674_path = process_pdf(claim.to_pdf(form_id: '21-674')) if claim.submittable_674?
-      form_686c_path = process_pdf(claim.to_pdf(form_id: '686C-674')) if claim.submittable_686?
+      form_674_path = process_pdf(claim.to_pdf(form_id: '21-674'), claim.created_at, '21-674') if claim.submittable_674?
+      form_686c_path = process_pdf(claim.to_pdf(form_id: '686C-674'), claim.created_at, '686C-674') if claim.submittable_686? # rubocop:disable Layout/LineLength
```

Comment:
Does Layout/LineLength need to be re-enabled after this?

Reply:
I think it can be disabled inline without needing to re-enable. I can wrap it if needed though

Reply:
That is my understanding as well based on documentation

https://docs.rubocop.org/rubocop/configuration.html#disabling-cops-within-source-code
![image](https://github.com/department-of-veterans-affairs/vets-api/assets/7817263/4dc53f8a-a630-4428-8bea-40e7be28cdb5)